h2o-algos/src/main/java/hex/glm/GLMModel.java

```diff
@@ -816,6 +816,7 @@ public void setZValues(double [] zValues, double dispersion, boolean dispersionE
     double [][] _vcov;
     private double _dispersion;
     private boolean _dispersionEstimated;
+    public int _responseColumn=-1; // represent response column index, only >= 0 when nclass > 2
```

Comment:
I think this is a little misleading, it is not natural to expect that _responseColumn would be populated only for multiclass problems.

Reply:
Removed .